integration_tests/e2e/workAndSkillsPage.cy.ts

```diff
@@ -188,25 +188,25 @@ context('Work and skills page', () => {
           workAndSkillsPage.GoalsEmploymentLabel().contains('Employment goals')
 
           workAndSkillsPage.GoalsEmploymentText().should('exist')
-          workAndSkillsPage.GoalsEmploymentText().contains('string')
+          workAndSkillsPage.GoalsEmploymentText().contains('An employment goal')
```

Comment:
The test data mocks/builders have been improved, in that they now return arrays containing reasonably sensible goal titles (rather than arrays of the word `string` as before)